### PR TITLE
feat: page scrolling

### PR DESCRIPTION
**Pull request by mdanilowicz**
### Description

Add smooth scrolling on the routing change (excluding product variants)

### CODE REVIEW

File: templates/vue-demo-store/app/router.options.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import type { RouterOptions } from "@nuxt/schema";
+
+export default <RouterOptions>{
+  scrollBehavior(to, from, savedPosition) {
+    /**
+     * Do not scroll to top when switching between product variants
+     *
+     * Assumptions:
+     * - product with variants contains minimum 2 subdirectories
```
Comment (mkucmus) [inline templates/vue-demo-store/app/router.options.ts:9]: we should add some more info (docs or here or both?) that the format of URL is not fixed - can be configured and then, the condition won't work. 
image

**Review by mkucmus — CHANGES_REQUESTED** [review]
smooth routing is what we love 💙 

please add one info about the URL format - If I understand it correctly

**Review by kstala — APPROVED** [review]

**Review by mkucmus — APPROVED** [review]

File: .changeset/strange-pets-relax.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"vue-demo-store": patch
+---
+
+- Add smooth scrolling on the routing change (excluding product variants)
+- Add smooth scrolling for listing pagination
```
Comment (patzick) [inline .changeset/strange-pets-relax.md:6]: that's another changeset entry in cms-base package

File: .changeset/strange-pets-relax.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+"vue-demo-store": patch
+---
+
+- Add smooth scrolling on the routing change (excluding product variants)
```
Comment (patzick) [inline .changeset/strange-pets-relax.md:5]:
```suggestion
Add smooth scrolling on the routing change (excluding product variants)
```

using `-` causes ugly changeset entry, like here:
image

**Review by patzick — CHANGES_REQUESTED** [review]

File: .changeset/strange-pets-relax.md (hunk 3 above)
Comment (patzick) [reply .changeset/strange-pets-relax.md]: yes, pure string without `-` is correct :)

**Review by patzick — APPROVED** [review]

File: .changeset/strange-pets-relax.md (hunk 3 above)
Comment (mdanilowicz) [reply .changeset/strange-pets-relax.md]: Ahh :D @patzick made me realise that it was a diff `+` ... ups 🙃

**Pull request merged**